### PR TITLE
Np 4995/file labeli18n

### PR DESCRIPTION
**Pull request by adamfox-nanopay**
ChoiceView has an issue between Edit mode and View mode.  The selectSpec uses foam.u2.tag.Select which manages translations based on locale in Edit mode, however in View mode the original text is always rendered regardless of locale.

### CODE REVIEW

**Review by tharmaman — APPROVED** [review]

**Review by lchanmann — APPROVED** [review]

File: src/foam/u2/view/ChoiceView.js [hunk 1]
```diff
@@ -238,7 +238,7 @@ foam.CLASS({
             .end();
         }
 
-        return self.E().add(self.text$);
+        return self.E().translate(self.text + ".name", self.text);
```
Comment (jlhughes) [inline src/foam/u2/view/ChoiceView.js:241]: What is 'name' at this point? Asking I expect to be translating the 'label'.

Comment (tharmaman) [reply src/foam/u2/view/ChoiceView.js:241]: According to docs, "this.text is the current label, this.label is the label for the whole view (eg. "Medal Color", not "Gold")."

RE: https://github.com/kgrgreer/foam3/pull/659/files#diff-eccdd95b711bd341a4fe353c6d21c0e1fdc2757568deae2515ae25c1b6d52449R32

Comment (lchanmann) [reply src/foam/u2/view/ChoiceView.js:241]: for translate(source, defaultValue) method call, the `source` is the choice view text + ".name" to be consistent with the Select tag, the `defaultValue` is the choice view text itself if it doesn't have translation.

Comment (kgrgreer) [reply src/foam/u2/view/ChoiceView.js:241]: .name is compatible with the Select in RW mode, so is correct.

**Pull request merged**